zipkin-lens/src/components/Browser/BrowserResults.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import BrowserResults from './BrowserResults';
+import TraceSummary from './TraceSummary';
+import { sortingMethods } from './sorting';
+
+describe('<BrowserResults />', () => {
+  const traceSummaryCommonProps = {
+    durationStr: 'μs', // dummy
+    timestamp: 1,
+    serviceSummaries: [],
+    spanCount: 0,
+    width: 1,
+  };
+  const defaultProps = {
+    traceSummaries: [
+      { traceId: '1', duration: 5, ...traceSummaryCommonProps },
+      { traceId: '2', duration: 3, ...traceSummaryCommonProps },
+      { traceId: '3', duration: 7, ...traceSummaryCommonProps },
+    ],
+    sortingMethod: sortingMethods.LONGEST,
+    skewCorrectedTracesMap: {
```

Comment:
this highlights to me if "skewCorrectedTracesMap" is a great name or not. clock skew is only one type of adjustment/correction. It might be helped to call this plainly tracesMap and differentiate something uncorrected as "raw traces"

Reply:
I see. I'll rename them 👍